packages/gtk3.rb

```diff
@@ -0,0 +1,36 @@
+require 'package'
+
+class Gtk3 < Package
+  description 'Gtk3 is a cross-platform widget toolkit for creating graphical user interfaces.'
+  homepage 'https://developer.gnome.org/gtk3/3.0/'
+  version '3.26'
+  source_url 'http://ftp.gnome.org/pub/gnome/sources/gtk+/3.22/gtk+-3.22.26.tar.xz'
+  source_sha256 '61eef0d320e541976e2dfe445729f12b5ade53050ee9de6184235cb60cd4b967'
+
+  depends_on 'glib'
+  depends_on 'gobject_introspection'
+  depends_on 'gdk_pixbuf'
+  depends_on 'cairo'
+  depends_on 'pango'
+  depends_on 'libepoxy'
+  depends_on 'graphene'
+  depends_on 'atk'
+  depends_on 'libxkbcommon'
+  depends_on 'meson'
+  depends_on 'ninja'
+  depends_on 'python3'
```

Comment:
Since python3 is already a dependency of ninja, it doesn't need to be included here.

Reply:
@uberhacker:
how do you keep the list of dependency, do you have a excel thing/or a script to know, what has 'already' been done/compiled.

Reply:
Before you include a dependency, you should drill down and find if it's already included in another dependency.  Obviously, this is easier when you have fewer dependencies.

Reply:
I should point out also that you want to avoid dependency loops.  Currently, both the graphene and gdk_pixbuf packages have dependency loops which prevent them from compiling so it's important that we only include the dependencies that are actually needed in a package.

Reply:
Yeah, the dependency loops should be avoided. The reason why I add all of them is just to show which package should be installed.